index.html

```diff
@@ -5,12 +5,138 @@
+      </div>
+  </div>
+</header>
+<section id="mainProgram">
+  <h2 class="mainTitle">Main Program</h2>
+  <div id="programs"></div>
+  <div class="btnContainer">
+      <button type="submit" class="mobileBtn" id="programBtn">Join Developer Best Practice Conference <br>
+          2022</button>
+  </div>
+  <h3 id="seeProgram" class="hide"><a href="#">SEE THE WHOLE PROGRAMM</a></h3>
+</section>
+<section id="speakers">
+  <div class="speakerTitleContainer">
+      <h2 class="speakersTitle">Featured Speakers</h2>
+  </div>
+  <div id="speakerResume">
+      <div id="exponentSample">
+          <div class="expositor" id="speaker1">
+              <div class="expositorImg"><img src="img/speakers/speaker_01.png" alt="expositor image"></div>
+              <div class="expositorTxt">
+                  <h2 class="expositorH2">Yochai Benkler</h2>
+                  <h3 class="expositorH3">Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School
+                  </h3>
+                  <p class="expositorP">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+              </div>
+          </div>
+          <div class="expositor" id="speaker3">
+              <div class="expositorImg"><img src="img/speakers/speaker_03.png" alt="expositor image"></div>
+              <div class="expositorTxt">
+                  <h2 class="expositorH2">SohYeong Noh</h2>
+                  <h3 class="expositorH3">Director of Art Centre Nabi and a board member of CC Korea</h3>
+                  <p class="expositorP">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+              </div>
+          </div>
+      </div>
+      <div id="exponents" class="hideExponents"></div>
+  </div>
+  <button type="submit" id="seeMore" class="seeMore">MORE <img src="img/buttons/arrow_down.png"
+          alt="Down pointer arrow"></button>
+</section>
+<section id="partners">
+  <h2 id='partnersh2'>Partner</h2>
+  <div id='partnersimgcont'>
+      <img class='partnersimg' src="img/logos/mozilla.png" alt="mozilla logo">
+      <img class='partnersimg' src="img/logos/airbnb.png" alt="air bnd logo">
+      <img src="img/logos/g.png" class='partnersimg' alt="google logo">
+      <img src="img/logos/naver.png" alt="naver logo" class='partnersimg'>
+      <img src="img/logos/daumkakao.png" alt="" class='partnersimg'>
+  </div>
+</section>
+<footer id="copyRight">  
+  <div id="crLogo">
+    <a href="index.html"><img id="crLogoimg" src="img/logos/icon.png" alt="3d systems logo"></a>
+    <p  class="crTxtLogo">Developers<br>Summit 2022</p>
+  </div> 
+  <div id="crTxt">
+    <h3 class="crtTxt">3D Systems 2022</h3>
+    <p class="crtTxt hideAll">freedom-respecting software</p>
+    <p class="crtTxt hideText">freedom-respecting software Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Id velit ut tortor pretium viverra suspendisse potenti nullam ac. Leo in vitae turpis massa.</p>
```

Comment:
- [OPTIONAL] Please avoid lines of code that are too long (100 characters or more). Makes your code cleaner and more readable. Follow the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). As not required by the previous reviewer I consider this optional.